app/assets/javascripts/organisation-autocomplete.js

```diff
@@ -5,16 +5,118 @@ window.GOVUKAdmin.Modules = window.GOVUKAdmin.Modules || {};
 
   Modules.OrganisationAutocomplete = function () {
     this.start = function ($element) {
-      accessibleAutocomplete.enhanceSelectElement({
-        selectElement: $element.find('#organisations')[0],
-        showAllValues: true,
-        autoselect: false,
-        dropdownArrow: dropdownArrow
-      });
-    };
 
-    function dropdownArrow() {
-      return '<span class="caret"></span>';
-    }
+      var autocompleteTemplateHtml = createAutocompleteTemplate();
+
+      $element
+        .append(addOrganisationButton())
+        .find('[selected]')
+        .each(function (index, option) {
+          if (index > 0) {
+            $(autocompleteTemplateHtml)
+              .insertAfter(lastSelectWrapper())
+              .find('option[value="' + $(option).attr('value') + '"]')
+              .attr('selected', 'selected')
+          }
+        });
+
+      enableAutocomplete();
+
+      function createAutocompleteTemplate() {
+        var $select = organisationSelects().first();
+        $select.removeAttr('multiple');
+        var $wrapper = wrapper($select);
+        var $template = $wrapper.clone();
+        $template.find('[selected]').removeAttr('selected');
+        return $template.prop('outerHTML');
+      }
+
+      function addOrganisationButton() {
+        return $(
+          '<button type="button" ' +
+          '        id="add-organisation" ' +
+          '        class="add-organisation js-add-organisation">' +
+          '  Add another organisation' +
+          '</button>'
+        ).click(addOrganisation);
+      }
+
+      function removeOrganisationButton() {
```

Comment:
As I commented above, I would prefer not to have HTML embedded in the JS if possible.